app/models/miq_product_feature/seeding.rb

```diff
@@ -0,0 +1,30 @@
+class MiqProductFeature < ApplicationRecord
+  module Seeding
+    extend ActiveSupport::Concern
+
+    RELATIVE_FIXTURE_PATH = "db/fixtures/miq_product_features".freeze
+    FIXTURE_PATH = Rails.root.join(RELATIVE_FIXTURE_PATH).freeze
+
+    module ClassMethods
+      def seed_files
+        return seed_root_filename, seed_core_files + seed_plugin_files
```

Comment:
@Fryguy If I'm reading correctly, this doesn't break the UI plugin work by @martinpovolny  because they should still be picked up in `seed_core_file`, right?  Note, I'm not sure if we still need it but I'm making sure that I understand it.  It looks to be still be intact.

Reply:
Correct, this keeps @martinpovolny 's original "directory of feature yamls" in core, however that's not being used anywhere, so while we have it we may want to consider dropping it.  On the plugin side though it could be useful to have separate yamls per feature in a dir, so I basically made plugins work like core.

Reply:
> have it we may want to consider dropping it

Yes, I'd like to see it dropped. Also it does not support any nesting if I remember correctly.

Someone might me using it: https://documentation.commvault.com/commvault/v11_adminconsole/article?p=87519.htm and they should be provided with a way to migrate.

What we need with this new way is documentation.